spec/xpm_ruby/connection_spec.rb

```diff
@@ -2,23 +2,17 @@
 
 module XpmRuby
   RSpec.describe(Connection) do
-    subject(:service) { described_class }
```

Comment:
Not super fussed if we drop this pattern, but I am curious if it was a conscious decision or not. 
;)

Reply:
Yeah I've thought about this pattern too... I sometimes remove it unconsciously to help me see all the code in one place rather than having to scroll around. My personal preference is readability over DRY in specs but YMMV?

[This article](http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/)seems to have stuck with me over the years.